src/app/docs/checkbox/checkbox-docs.component.ts

```diff
@@ -6,4 +6,41 @@ import {
   selector: 'app-checkbox-docs',
   templateUrl: './checkbox-docs.component.html'
 })
-export class CheckboxDocsComponent { }
+export class CheckboxDocsComponent {
+
+  public checkboxItems: any[] = [
+    {
+      label: 'Selected checkbox',
```

Comment:
Same note as on the radio button demo about how the first two labels become inaccurate when users play around with the demo. Can we just follow the pattern you switched to there and label these checkboxes `Checkbox 1`, `Checkbox 2`, and `Checkbox 3 is disabled`?

Reply:
That said, if we do that, the `Checkbox states` header should probably also change to `Standard checkboxes`.

Reply:
Yep, good call. Done.